pkg/service/v1alpha1/controller.go

```diff
@@ -118,7 +125,9 @@ func (cs *controller) CreateVolume(
 	if err == nil && cvc != nil && cvc.DeletionTimestamp == nil {
 		goto createVolumeResponse
 	}
-	err = utils.ProvisionVolume(size, volName, rCount, cspcName, snapshotID, policyName)
+	err = utils.ProvisionVolume(size, volName, rCount,
+		cspcName, snapshotID,
+		nodeID, policyName)
```

Comment:
to many args , better to have a struct with above options called `volumeOptions` populate it and use it as argument to the func

Reply:
we can fix this upcomig PR as well